tailwind.config.js

```diff
@@ -4,14 +4,14 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        rent: "#BC87FF",
-        transfer: "#88C6FF",
-        administration: "#E92FAC",
-        eligibility: "#EC3E55",
-        security: "#FFA500",
-        develop: "#49C6A6",
-        stewardship: "#9FF876",
-        use: "#FEFC41",
+        rent: 'rgb(var(--color-rent) / <alpha-value>)',
+        transfer: 'rgb(var(--color-transfer) / <alpha-value>)',
+        administration: 'rgb(var(--color-administration) / <alpha-value>)',
+        eligibility: 'rgb(var(--color-eligibility) / <alpha-value>)',
+        security: 'rgb(var(--color-security) / <alpha-value>)',
+        develop: 'rgb(var(--color-develop) / <alpha-value>)',
+        stewardship: 'rgb(var(--color-stewardship) / <alpha-value>)',
+        use: 'rgb(var(--color-use) / <alpha-value>)',
```

Comment:
just making sure i understand this: this change now gives us option to dynamically pass an opacity value? nice!

Reply:
https://tailwindcss.com/docs/customizing-colors#using-css-variables

Yep, defining these colours via their RGB colours allow them to behave more like Tailwind colours should 👌